spec/factories/events/pull_requests.rb

```diff
@@ -53,5 +53,26 @@
     repository
 
     association :owner, factory: :user
+
+    trait :merged do
+      state { 'closed' }
+      merged_at { Time.current }
+
+      transient do
+        merge_time { (merged_at - opened_at).to_i }
+      end
+
+      after(:create) do |pull_request, evaluator|
+        create(
+          :merge_time,
+          pull_request: pull_request,
+          value: evaluator.merge_time
+        )
+        create(
+          :review_coverage,
+          pull_request: pull_request
+        )
+      end
```

Comment:
I think this should be created manually in each test that requires it. Maybe using a trait isn’t too bad, as long as we don’t overuse it, wdyt?

Reply:
Given that this logic is used in many tests, I thought it was a good idea to have the trait to avoid repeating all the logic. Also, it's a simple logic and not a complex trait. Do you think it's ok in this case?